packages/vapor/scss/components/badge.scss

```diff
@@ -72,6 +72,15 @@
         --background-color: var(--sienna-orange-20);
     }
 
+    &.mod-tag {
+        background: var(--white);
+        font-weight: var(--main-font-bold);
+    }
+
+    &.mod-no-capitalize {
+        text-transform: inherit;
+    }
```

Comment:
I makes sense to have all badges not capitalized by default IMO

Reply:
Isn't it a breaking change thought? :sweat: 

I'm fine if you are good with it

Reply:
It is, but I'm good with it. Worse case we will have some capital letters to add in some places.